src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -64,7 +64,10 @@ The .NET Foundation licenses this file to you under the MIT license.
       <NetCoreAppNativeLibrary Include="@(NetCoreAppNativeLibrary->'%(Identity)')">
         <EscapedPath>$(IlcFrameworkNativePath)lib%(Identity).a</EscapedPath>
       </NetCoreAppNativeLibrary>
+      <NativeLibrary Include="$(NugetPackageRoot)/microsoft.dotnet.ilcompiler/7.0.0-dev/shim_source_code/libs/System.Globalization.Native/build/libSystem.Globalization.Native-Static.a" Condition="'$(LocalSystemGlobalizationNative)' != ''"/>
       <NativeLibrary Include="@(NetCoreAppNativeLibrary->'%(EscapedPath)')" />
+      <StaticICULibs Include="/usr/lib/x86_64-linux-gnu/libicuio.a;/usr/lib/x86_64-linux-gnu/libicutu.a;/usr/lib/x86_64-linux-gnu/libicui18n.a;/usr/lib/x86_64-linux-gnu/libicuuc.a;/usr/lib/x86_64-linux-gnu/libicudata.a" Condition="'$(LocalSystemGlobalizationNative)' != '' And Exists('/usr/lib/x86_64-linux-gnu/libicuio.a')" />
```

Comment:
These hardcoded paths with x86_64 need to be fixed too.

Reply:
Addressed in next revision